src/main/java/org/owasp/esapi/logging/java/ESAPIErrorJavaLevel.java

```diff
@@ -27,8 +29,10 @@
     /**
      * Defines a custom error level below SEVERE but above WARNING since this level isn't defined directly
      * by java.util.Logger already.
+     * @deprecated
+     * @see ESAPICustomJavaLevel#ERROR_LEVEL
      */
-    public static final Level ERROR_LEVEL = new ESAPIErrorJavaLevel( "ERROR", Level.SEVERE.intValue() - 1);
+    public static final Level ERROR_LEVEL = new ESAPIErrorJavaLevel( ESAPICustomJavaLevel.ERROR_LEVEL.getName(),ESAPICustomJavaLevel.ERROR_LEVEL.intValue());
```

Comment:
This update ensures that both values remain consistent until this class is deleted.  I verified that the enumeration constructor is using the same values, and this should not impact any existing functionality.